docs/mp/oci/01_oci.adoc

```diff
@@ -0,0 +1,71 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2021 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+:javadoc-base-url-api: {javadoc-base-url}io.helidon.config/io/helidon/oci
+
+= Oracle Cloud Infrastructure Integration
+:h1Prefix: MP
+:description: Helidon OCI Integration
+:keywords: oci
+:common-deps-page-prefix-inc: ../../shared/dependencies/common_shared.adoc
+:feature-name: OCI Integration
+
+The Helidon MP OCI provides easy access to Oracle Cloud Infrastructure.
+
+In order to be used correctly in Helidon MP CDI environment the following maven dependency should be included as a prerequisite:
+
+include::{common-deps-page-prefix-inc}[tag=maven-dependency]
+
+[source,xml]
+----
+<dependency>
+    <groupId>io.helidon.integrations.oci</groupId>
+    <artifactId>helidon-integrations-oci-cdi</artifactId>
+</dependency>
+----
+
+== General configuration
+
+=== Via Helidon Properties configuration
```

Comment:
Using the Helidon properties configuration:

Reply:
fixed